pandapower/test/control/test_tap_dependent_impedance.py

```diff
@@ -12,6 +12,7 @@
     trafo_characteristics_diagnostic
 
 
+# todo add tests for voltage_ratio and angle_dev?
```

Comment:
yes :-)

Reply:
leaving as todo for now - will add later